tests/test_device_models.py

```
@@ -1,182 +1,197 @@
+            {"ip": "192.168.1.1", "hostname": "ap-1"},
+            {"ip": "192.168.1.1", "hostname": "ap-1"},
+        ),
+        (
+            "timing_fields",
+            {"uptime": 3600, "last_seen": 1234567890},
+            {"uptime": 3600, "last_seen": 1234567890},
+        ),
+    ],
+)
+def test_device_model_fields(
+    field_group: str, values: Dict[str, Any], expected_values: Dict[str, Any]
```

Ruff (0.8.2)

Unused function argument: `field_group`

(ARG001)